src/App.js

```diff
@@ -1,13 +1,30 @@
 /* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] You have **disabled linter checks** on your code. Please enable **all** linter check on your code to ensure that your code is robust code and clean.